media_service/mediastore.py

```diff
@@ -24,6 +27,30 @@
 class MediaStoreUploadException(Exception):
     pass
 
+def process_zip(filelist):
+    '''
+    processes a file upload list, unzipping all zips
+    returns a new list with unzipped files
+    '''
+    newlist = []
+    for file in filelist:
+        if "zip" in file.content_type:
```

Comment:
Are we assuming that the MIME type will always be `application/zip` for a valid ZIP file? I'm not sure if that's a valid assumption or not. 

Just curious, but would [python-magic](https://github.com/ahupp/python-magic) be more effective in detecting that the file is a ZIP file? We're [already using python-magic](https://github.com/Harvard-ATG/media_management_api/blob/master/media_service/mediastore.py#L213) to detect image file types, so it should be available if we wanted to use it here too.